pkg/users/client.go

```diff
@@ -160,7 +161,8 @@ type AuthenticateWithMagicAuthOpts struct {
 }
 
 type AuthenticationResponse struct {
```

Comment:
In ruby I think this is called `AuthenticationFactorAndChallenge` (or was 6 days ago, we may want to discuss standardization on this)